connectors-home/modules/ROOT/pages/introduction/connectors-shared-responsibilities.adoc

```diff
@@ -0,0 +1,90 @@
+
+= Anypoint Connectors Shared Responsibilities
+
+The successful operation of Anypoint Connectors is a shared responsibility between MuleSoft and the customer. It is critical to understand which areas customers are responsible for and which areas MuleSoft is responsible for.
+
+== Anypoint Connectors Categories
+
+Anypoint Connectors fall into these categories:
+
+* Connectors that enable the use of cross-platform protocols or open APIs. +
+For example, Database Connector, which is based on Oracle’s JDBC protocol.
```

Comment:
I would leave a blank line between the two sentences here and elsewhere. It makes it easier for folks to skim.